dm/dm/proto/dmworker.proto

```diff
@@ -202,6 +202,19 @@ message SubTaskStatus {
         LoadStatus load = 9;
         SyncStatus sync = 10;
     }
+    ValidationBriefStatus validation = 11;
+}
+
+message ValidationBriefStatus {
+    string mode = 1;
+    Stage status = 2;
+    ValidationStats stats = 3;
+}
+
+message ValidationStats {
+    string ProcessedRows = 1;
+    string PendingRows = 2;
+    string ErrorRows = 3;
```

Comment:
remove those and use ValidationStatus instead

Reply:
It seems ValidationStatus includes some superfluous fields, which will be printed as a result of `query-status` (but it does make code simpler, I will discuss with PM later)

Reply:
better have a sample output so we can compare